#ifndef LOCAL_PATH_PLANNER_H_
#define LOCAL_PATH_PLANNER_H_

#include "rclcpp/rclcpp.hpp"
#include "config_reader.h"
#include "curve.h"
#include "polynomial_curve.h"
#include "main_car_info.h"
#include "decision_center.h"
#include "local_path_smoother.h"

namespace Planning
{
  class LocalPathPlanner  // 局部路径规划器
  {
  public:
    LocalPathPlanner();
  };

} // namespace Planning
#endif // LOCAL_PATH_PLANNER_H_